### PR TITLE
docs: Added "dagger run" to get started guides

### PR DESCRIPTION
**Pull request by vikram-dagger**
This commit updates the Go, Node.js, Python and CLI get started guides to use "dagger run".

Additional PRs should be expected for other sections of the docs.

### CODE REVIEW

File: docs/current/sdk/python/628797-get-started.md [hunk 1]
```diff
@@ -101,7 +102,7 @@ The `from_()`, `with_directory()`, `with_workdir()` and `with_exec()` methods al
 Run the Python CI tool by executing the command below:
 
 ```shell
-python test.py
+dagger run python test.py
```
Comment (vikram-dagger) [inline docs/current/sdk/python/628797-get-started.md:105]: @helderco this and subsequent examples fails with the fastapi repo. The error seen is below. Can you please check and let me know what changes are needed to the examples?

```
21: [15.1s]
21: [15.1s] [notice] A new release of pip is available: 23.0.1 -> 23.2.1
21: [15.1s] [notice] To update, run: pip install --upgrade pip
21: exec pip install -e .[test] DONE

20: exec pytest tests
20: [0.80s] panic: exec: "pytest": executable file not found in $PATH
20: [0.80s]
20: [0.80s] goroutine 1 [running, locked to thread]:
20: [0.80s] main.shim()
20: [0.80s]     /app/cmd/shim/main.go:267 +0xfd8
20: [0.80s] main.main()
20: [0.80s]     /app/cmd/shim/main.go:64 +0x5d
20: exec pytest tests ERROR: process "pytest tests" did not complete successfully: exit code: 2
Traceback (most recent call last):
```

Comment (helderco) [reply docs/current/sdk/python/628797-get-started.md:105]: They changed the way they install test dependencies in development. We'll need to change this:

```diff
-                .with_exec(["pip", "install", "-e", ".[test]"])
+                .with_exec(["pip", "install", "-r", "requirements.txt"])
```
Probably better to pin a tag in the clone or something.

Comment (vikram-dagger) [reply docs/current/sdk/python/628797-get-started.md:105]: Modified

**Review by TomChv — APPROVED** [review]
LGTM on NodeJS side

**Pull request merged**